Generate.py

```diff
@@ -454,6 +454,8 @@ def handle_option(ret: argparse.Namespace, game_weights: dict, option_key: str,
             raise Exception(f"Error generating option {option_key} in {ret.game}") from e
         else:
             player_option.verify(AutoWorldRegister.world_types[ret.game], ret.name, plando_options)
+    elif option.default == "random":
+        setattr(ret, option_key, option.from_any(option.default))
     else:
         setattr(ret, option_key, option(option.default))
```

Comment:
I wondering if this can be dropped - i.e. always go through `from_any(default)` and add a comment `# from_any required for default = random`

Reply:
honestly i thought i had tried that but apparently not lmao